draft-ietf-tsvwg-dtls-over-sctp-bis.md

```diff
@@ -921,34 +938,33 @@ normative:
    user messages and ensures that all protected user message data has
    been successfully transferred to the remote ULP.
 
-   4. Local DTLS/SCTP sends DTLS Close_notify to remote instance of
-   DTLS/SCTP on each and all DTLS connections, keys and session
-   state are kept for processing packets received later on.
+   4. Local DTLS/SCTP sends a DTLS/SCTP Controll Message
+   {{Control-Message}} of type "SHUTDOWN_Request" {{SHUTDOWN-Request}}
+   to its peer.
 
-   5. When receiving Close_notify on the last open DTLS connection,
-   remote DTLS/SCTP instance informs its ULP that remote shutdown has
-   been initiated. When two parallel DTLS connections are in place it
-   is important to await Close_notify alert on both to not misstake a
-   rekeying. No more ULP user message data to be sent to peer can be
-   accepted by DTLS/SCTP. In case this endpoint has initiated and DTLS
+   5. When receiving the SHUTDOWN-Request the remote DTLS/SCTP
+   instance informs its ULP that remote shutdown has been
+   initiated. No more ULP user message data to be sent to peer can be
+   accepted by DTLS/SCTP. In case this endpoint has initiated a DTLS
    connection handshake this MUST be aborted as the peer is unable to
-   respond.
+   respond to avoid additional case of draining.
 
    6. Remote DTLS/SCTP finishes any protection operation on buffered
    user messages and ensures that all protected user message data has
    been successfully transferred to the remote ULP.
 
-   7. Remote DTLS/SCTP sends Close_notify to Local DTLS/SCTP entity
-   for each and all DTLS connections.
+   7. Remote DTLS/SCTP sends DTLS Close_notify to Local DTLS/SCTP
+   entity for each and all DTLS connections. Then it initiates the
+   SCTP shutdown procedure (section 9.2 of {{RFC4960}}).
```

Comment:
RFC9260

Reply:
Lets do that change is a separate PR across the whole document.